nntrainer/tensor/tensor.cpp

```diff
@@ -92,6 +92,35 @@ Tensor::Tensor(const TensorDim &d, const float *buf) : Tensor() {
   }
 }
 
+Tensor Tensor::Wrap(float *buf, const TensorDim &d, int offset) {
+  if (d.getDataLen() == 0 || buf == nullptr) {
```

Comment:
It more like map rather than wrap. I guess.

Reply:
I feel like map implies one to one mapping. wrap implies it wraps a memory with a tensor to use tensor methods. 

Thanks for the suggestion :) @kparichay Could you give an opinion as well?

Reply:
Changed to Tensor::Map :)